packages/live-share-media/src/MediaPlayerSynchronizer.ts

```diff
@@ -630,7 +663,7 @@ export class MediaPlayerSynchronizer extends EventEmitter {
                     this.emit(MediaPlayerSynchronizerEvents.groupaction, {
                         type: MediaPlayerSynchronizerEvents.groupaction,
                         details: details,
-                        playerError: error,
+                        error,
```

Comment:
This fixes the naming, which did not match the IMediaPlayerSynchronizerEvent interface. This code path is also not used.

Reply:
I agree this is better naming...but wouldn't this be a breaking change for apps using this? We should be careful here...I think the YouTube app uses this field, for example.

This might need to be marked as a change we make in v2.0.

Reply:
but this code path is not used?